Ukoly/DU_9/class.py

```diff
@@ -0,0 +1,64 @@
+class band_member:
```

Comment:
V Pythone sa na pomenovávanie tried používa _PascalCase_... Teda prvé písmenko je veľké, po ňom náledujú ďalšie malé, okrem prípadu, ak sa jedná o viacero-slovné (v iných prípadoch tam používaš '_'), vtedy ďalšie slová opäť začínaš veľkými písmenkami, teda v tomto prípade `BandMember`.